### PR TITLE
base HTML

### PR DESCRIPTION
**Pull request by adamjfield**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,128 @@
...
+        <select name="meal-type" class="select-dropdown">
+          <option value="" disabled selected>Pick A Meal Type</option>
+          <option value="breakfast">Breakfast</option>
+          <option value="lunch">Lunch</option>
+          <option value="dinner">Dinner</option>
+          <option value="snack">Snack</option>
+        </select>
+        <select name="movie-type" class="select-dropdown">
+          <option value="" disabled selected>Pick A Movie Genre</option>
+          <option value="action">Action</option>
```
Comment (baxters4karma) [inline index.html:32]: The Action through Horror option elements can be removed as the movies.js code will dynamically create by fetching data from movie API.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,128 @@
...
+      <div>
+        <h2>Make Your Selections!</h2>
+        <select name="meal-type" class="select-dropdown">
+          <option value="" disabled selected>Pick A Meal Type</option>
+          <option value="breakfast">Breakfast</option>
+          <option value="lunch">Lunch</option>
+          <option value="dinner">Dinner</option>
+          <option value="snack">Snack</option>
+        </select>
+        <select name="movie-type" class="select-dropdown">
```
Comment (baxters4karma) [inline index.html:30]: Please add......id="movie-type"........to this select so the populateGenreDropDown function in movies.js will pull correct DOM element. Thank you!

File: index.html [hunk 3]
```diff
@@ -1,12 +1,128 @@
...
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
```
Comment (baxters4karma) [inline index.html:7]: If the <title> element was removed from <head>, we should add back as follows so tab of web page will have title:  
<title>Looking for Dinner and Movie</title>. If this is being handled differently, please just let me know that.

File: index.html [hunk 4]
```diff
@@ -1,12 +1,128 @@
...
+        reiciendis aspernatur saepe aliquid commodi iusto aliquam quis ad amet
+        libero.
+      </p>
+    </header>
+    <body>
+      <!-- search form start -->
+      <div>
+        <h2>Make Your Selections!</h2>
+        <select name="meal-type" class="select-dropdown">
+          <option value="" disabled selected>Pick A Meal Type</option>
```
Comment (baxters4karma) [inline index.html:24]: What do you guys think about changing "Pick A Meal Type" to "Choose a Meal Type..."?

File: index.html [hunk 5]
```diff
@@ -1,12 +1,128 @@
...
+          <option value="action">Action</option>
+          <option value="comedy">Comedy</option>
+          <option value="romance">Romance</option>
+          <option value="thriller">Thriller</option>
+          <option value="horror">Horror</option>
+        </select>
+        <button>Submit</button>
+      </div>
+      <!-- Search form end -->
+      <!-- Previous search - test only - this will be dynamical generated.
```
Comment (baxters4karma) [inline index.html:41]: I am not sure how to view this updated file within browser so did not review the previous search structure. Keeping as is for now and we can plan to review further.

**Review by baxters4karma — CHANGES_REQUESTED** [review]
Wasn't sure if I should choose Comment or Request changes, apologies if I chose the wrong one. If so, just let me know and I can see what I need to do to correct.

Nicely done, Adam.....coding is looking good! :-)

File: index.html (hunk 1 above)
Comment (adamjfield) [reply index.html:33]: Updated movie dropdown list to match

File: index.html (hunk 2 above)
Comment (adamjfield) [reply index.html]: Added ID to select element

File: index.html (hunk 4 above)
Comment (adamjfield) [reply index.html:25]: Didn't change this now but we can at any point

**Review by baxters4karma — APPROVED** [review]

**Pull request merged**